app/services/api/update_trainee_service/v01.rb

```diff
@@ -62,6 +71,12 @@ def hesa_trainee_detail_attributes_errors
       def validation
         @validation ||= Submissions::ApiTrnValidator.new(trainee:)
       end
+
+      def attribute_validation_errors
```

Comment:
This is called only at [line 21](https://github.com/DFE-Digital/register-trainee-teachers/pull/4273/files#diff-e17090611492f57c7ac8004bc34895c6285f2505fd4f811ab11bf3d3323a4292R21) in the commented out code.